mediumDifficultyAll Pro/Medium difficulty Program 1/taxSalary/taxSalary.java

```diff
@@ -0,0 +1,45 @@
+package taxSalary;
+
+class taxSalary {
+    /*
+    Q.1 Write a java program to find tax for employee. The program should
+    calculate the income tax to be paid by the employee as per the criteria given below:
+    Slab rate IT rate
+    Upto Rs. 7,00,000 Nil
+    Above Rs. 7,00,000 (0-5,00,000=nil,5,00,000 – 7,00,000=10%)
+    Rs. 7,00,000-10,00,000 15% on additional amount
+    Rs. 10,00,000-15,00,000 20% on additional amount
+
+    Above 15,00,000 30% on additional amount
+    Result: - income tax is …………………………….
+    */
+    public static void main(String[] args) {
+        taxSalaryCount(1000000);
+    }
+    public static void taxSalaryCount(double salary) {
+        double tax = 0;
```

Comment:
why using double? Practically salary will be in int. If the  calculated tax is in double format use rounding off.